lib/autoloads/import_csv.rb

```diff
@@ -0,0 +1,21 @@
+class ImportCsv
+  def self.import(path)
+    list = []
+    CSV.foreach(path, headers: true) do |row|
+      data = row.to_h.transform_values do |val|
+        /^[0-9]+$/ =~ val ? val.to_i : val
```

Comment:
いいですね！こちらの形式にするとなおよいですよ。

```suggestion
        /\A\d+\z/ =~ val ? val.to_i : val
```

こちらの記事も参考にされるとよいですね。
https://qiita.com/jnchito/items/ea7832df6f64a9034872